rsmanage/rsmanage.py

```diff
@@ -51,7 +51,11 @@ def cli(config, verbose, if_clean):
     config.conf = conf
     config.dbname = re.match(r"postgres.*//.*?@.*?/(.*)", config.dburl).group(1)
     config.dbhost = re.match(r"postgres.*//.*?@(.*?)/(.*)", config.dburl).group(1)
-    config.dbuser = re.match(r"postgres.*//(.*?):.*?@(.*?)/(.*)", config.dburl).group(1)
+    if conf  != "production":
+        config.dbuser = re.match(r"postgres.*//(.*?)(:.*?)?@(.*?)/(.*)", config.dburl).group(1)
+    else:
+        config.dbuser = re.match(r"postgres.*//(.*?):(.*?)@(.*?)/(.*)", config.dburl).group(1)
```

Comment:
I don't really see what this accomplishes.  group(1) is still going to return the same thing in either case.  I can see that having the : inside the paren was probably wrong but not sure that changes anything in this case.